tests/migrate_from_0_1_0.rs

```diff
@@ -23,23 +20,33 @@ mod tests {
     const UOSMO_UATOM_PATH: &[(u64, &str, &str); 1] = &[(1, UOSMO, UATOM)];
     const UION_UATOM_PATH: &[(u64, &str, &str); 2] = &[(2, UION, UOSMO), (1, UOSMO, UATOM)];
 
-    fn osmosis_swap_operations_list_from_vec(vec: &[(u64, &str, &str)]) -> SwapOperationsList {
-        SwapOperationsList::new(
+    #[allow(deprecated)]
+    fn osmosis_swap_operations_list_from_vec(
+        vec: &[(u64, &str, &str)],
+    ) -> cw_dex_router_0_3::operations::SwapOperationsList {
```

Comment:
Needed to use old version? Nothing has changed in API?

Reply:
good point changed to new api and removed 0.3 dependency